app/sitemap.txt

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
I've found that some secondary search engines are more successful with `txt` version (Bing, Yahoo, etc ...) be good to keep both.